sdk/ai/azure-ai-projects/README.md

```diff
@@ -535,140 +535,164 @@ agent = await project_client.agents.create_agent(
 
 #### Create Agent With Azure Function Call
 
-The agent can handle Azure Function calls on the service side and return the result of the call. To use the function, we need to create the `AzureFunctionTool`, which contains the input and output queues of Azure function and the description of input parameters. Please note that in the prompt we are asking the model to invoke queue when the specific question ("What would foo say?") is being asked. See below for the instructions on function deployment.
+The AI agent leverages Azure Functions triggered asynchronously via Azure Storage Queues. To enable the agent to perform Azure Function calls, you must set up the corresponding `AzureFunctionTool`, specifying input and output queues as well as parameter definitions.
+
+Example Python snippet illustrating how you create an agent utilizing the Azure Function Tool:
 
 <!-- SNIPPET sample_agents_azure_functions.create_agent_with_azure_function_tool -->
 
 ```python
+storage_service_endpoint = "https://<your-storage>.queue.core.windows.net"
+
 azure_function_tool = AzureFunctionTool(
-    name="foo",
-    description="Get answers from the foo bot.",
+    name="get_weather",
+    description="Get weather information using Azure Function",
     parameters={
-        "type": "object",
-        "properties": {
-            "query": {"type": "string", "description": "The question to ask."},
-            "outputqueueuri": {"type": "string", "description": "The full output queue uri."},
+            "type": "object",
+            "properties": {
+                "location": {"type": "string", "description": "The location of the weather."},
+            },
+            "required": ["location"],
         },
-    },
     input_queue=AzureFunctionStorageQueue(
-        queue_name="azure-function-foo-input",
+        queue_name="input",
         storage_service_endpoint=storage_service_endpoint,
     ),
     output_queue=AzureFunctionStorageQueue(
-        queue_name="azure-function-tool-output",
+        queue_name="output",
         storage_service_endpoint=storage_service_endpoint,
     ),
 )
 
 agent = project_client.agents.create_agent(
-    model="gpt-4",
-    name="azure-function-agent-foo",
-    instructions=f"You are a helpful support agent. Use the provided function any time the prompt contains the string 'What would foo say?'. When you invoke the function, ALWAYS specify the output queue uri parameter as '{storage_service_endpoint}/azure-function-tool-output'. Always responds with \"Foo says\" and then the response from the tool.",
-    tools=azure_function_tool.definitions,
+    model=os.environ["MODEL_DEPLOYMENT_NAME"],
+    name="my-assistant",
+    instructions="You are a helpful assistant",
+    toolset=azure_function_tool.definitions,
```

Comment:
Should it ne tool_definition?

Reply:
thanks, fixed to

tools=azure_function_tool.definitions,